guides/custom_video.md

```diff
@@ -0,0 +1,215 @@
+
+## Prerequisites
+
+* You have read [Building a Meeting Application using the Amazon Chime SDK](https://aws.amazon.com/blogs/business-productivity/building-a-meeting-application-using-the-amazon-chime-sdk/). You understand the basic architecture of Amazon Chime SDK and deployed a serverless/browser demo meeting application.
+* You have a basic to intermediate understanding of iOS development and tools.
+* You have installed Xcode version 11.3 or later.
+* You have read the [API overview](https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/api_overview.md) and have a basic understanding of the components covered in that document.
+* You have completed [Getting Started](https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/getting_started.md) and have running application which uses the Amazon Chime SDK.
+
+Note: Deploying the serverless/browser demo and receiving traffic from the demo created in this post can incur AWS charges.
+
+## Using the provided camera capture implementation as a custom source to access additional functionality
+
+While the Amazon Chime SDK internally uses a implementation of camera capture, the same capturer can be created, maintained, and used externally before being passed in for transmission to remote participants using the [AudioVideoFacade](https://aws.github.io/amazon-chime-sdk-ios/Protocols.html#/c:@M@AmazonChimeSDK@objc(pl)AudioVideoFacade). This grants access to the following features:
+
+* Explicit camera device and format selection.
+* Configuration, starting, stopping, and video renderering before joining the call.
+* Torch/flashlight control.
+
+The camera capture implementation is found in [DefaultCameraCaptureSource](https://aws.github.io/amazon-chime-sdk-ios/Classes/DefaultCameraCaptureSource.html). To create and use the camera capture source complete the following steps:
+
+1. Create a [DefaultCameraCaptureSource](https://aws.github.io/amazon-chime-sdk-ios/Classes/DefaultCameraCaptureSource.html). This requires a [Logger](https://aws.github.io/amazon-chime-sdk-ios/Protocols/Logger.html) as dependency.
+
+```
+    let cameraCaptureSource = DefaultCameraCaptureSource(logger: logger)
+```
+
+3. Call [VideoCaptureSource.start()](Protocols/VideoCaptureSource.html#/c:@M@AmazonChimeSDK@objc(pl)VideoCaptureSource(im)start) and [DefaultCameraCaptureSource.stop()](https://aws.github.io/amazon-chime-sdk-ios/Classes/DefaultCameraCaptureSource.html#/c:@M@AmazonChimeSDK@objc(pl)VideoCaptureSource(im)stop) to start and stop the capture respectively. Note if no [VideoSink](https://aws.github.io/amazon-chime-sdk-ios/Protocols/VideoSink.html) have been attached (see later sections) that captured frames will be immediately dropped.
```

Comment:
same as android 

Note if no [VideoSink](https://aws.github.io/amazon-chime-sdk-ios/Protocols/VideoSink.html) have been attached (see later sections) that captured frames will be immediately dropped.
->
Note **that** if no [VideoSink](https://aws.github.io/amazon-chime-sdk-ios/Protocols/VideoSink.html) **has** been attached (see later sections) that captured frames will be immediately dropped.

Reply:
Ah, my bad for not fixing up both at same time.